health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/ShowMoreWrapper.js

```
@@ -0,0 +1,41 @@
+export const ShowMoreWrapper = ({ setShowPopUp, alreadyQueuedSelectedState, heading="" }) => {
+    const { t } = useTranslation();
+   
+    return (
+       <PopUp
+       className={""}
+       style={{
+           maxWidth: '40%'
+       }}
+       type={"default"}
+       heading={t(heading)}
+       children={[]}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)